### PR TITLE
style(frontend): new tertiary-link class for buttons [GIX-3048]

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We introduce a new class of buttons called `tertiary-link`. Its first usage is for the `ButtonIcon` components. 

### Normal state

Screenshot 2024-10-02 at 19 09 28

### Hover state

Screenshot 2024-10-02 at 19 09 35

### Usage with all states

https://github.com/user-attachments/assets/284bf65d-f9cc-4ecb-b230-cb3654c70bea

### CODE REVIEW

File: src/frontend/src/lib/components/ui/ButtonIcon.svelte [hunk 1]
```diff
@@ -5,14 +5,12 @@
 </script>
 
 <button
-	class="icon text-primary flex flex-col text-center text-xs font-normal"
+	class="tertiary-link icon flex flex-col text-center text-xs font-normal"
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/ui/ButtonIcon.svelte:8]: Curiosity question, why not just calling it "tertiary"? Why "-link"?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/ui/ButtonIcon.svelte:8]: According to plans and Figma, there is a "tertiary" class

Comment (peterpeterparker) [reply src/frontend/src/lib/components/ui/ButtonIcon.svelte:8]: Different than `tertiary-link`? I know it's minimal, just feel like that adding a suffix here feels like already adding an exception you know what I mean?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/ui/ButtonIcon.svelte:8]: understood! well the difference is really the color set ultimately

Do you prefer that I create a class `tertiary` with a sub class `link`

Comment (peterpeterparker) [reply src/frontend/src/lib/components/ui/ButtonIcon.svelte:8]: Oh yes, I would rather like that if it makes sense to you too?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/ui/ButtonIcon.svelte]: yes yes., done! have a look

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**